hack/update-generated-agent-proto.sh

```diff
@@ -3,4 +3,5 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+#!/bin/sh
```

Comment:
This doesn't do anything as it isn't the first line of the file.

Reply:
OK, will remove this soon.